tasks/broad/UltimaGenomicsGermlineFilteringThreshold.wdl

```diff
@@ -0,0 +1,388 @@
+    >>>
+    output {
+        File output_vcf_file = "~{output_vcf}"
+        File output_vcf_index = "~{output_vcf}.tbi"
+    }
+    runtime {
+        memory: "8GB"
+        disks: "local-disk " + (ceil(size(input_vcf, "GB")) * 3 + 10) + " HDD"
+        docker: docker
+        cpu: 1
+    }
+}
+
+task FilterSampleVCF{
+    input{
+        File input_vcf
+        String docker = "gcr.io/terra-project-249020/jukebox_vc:test_jc_optimize_3d7509"
+    }
+
+    String output_vcf = basename(input_vcf, ".vcf.gz") + ".filter_unused.vcf.gz"
+
+    command <<<
+        set -e
+        source ~/.bashrc
+        conda activate genomics.py3
+        bcftools view -a -i 'GT[*]="alt"' ~{input_vcf} -o ~{output_vcf} -O z 
+        bcftools index -t ~{output_vcf}
+    >>>
+
+    output {
+        File output_vcf_file = "~{output_vcf}"
+        File output_vcf_index = "~{output_vcf}.tbi"
+    }
+
+    runtime {
+        memory: "8GB"
+        disks: "local-disk " + (ceil(size(input_vcf, "GB")) *2 + 10) + " HDD"
+        docker: docker
+        cpu: 1
+    }
+}
+
+task FilterSymbolicAlleles {
+    input {
+        String base_file_name
+        File input_vcf
+        File input_vcf_index
+        String docker = "gcr.io/terra-project-249020/jukebox_vc:test_jc_optimize_3d7509"
+    }
+
+    String output_vcf_name = "~{base_file_name}" + ".vcf.gz"
+    command <<<
+        set -e
+        source ~/.bashrc
+        conda activate genomics.py3
+        gatk --java-options "-Xmx10g"  SelectVariants \
+            -V ~{input_vcf} \
+            -O ~{output_vcf_name}.tmp.vcf.gz \
+            --remove-unused-alternates
+        gatk --java-options "-Xmx10g" SelectVariants \
+            -V ~{output_vcf_name}.tmp.vcf.gz \
+            -O ~{output_vcf_name} \
+            --exclude-non-variants \
+            --select-type-to-exclude SYMBOLIC
+        >>>
+    runtime {
+        memory: "12 GB"
+        cpu: 1
+        disks: "local-disk " + (ceil(size(input_vcf, "GB")) *4 +10) + " HDD"
+        docker: docker
+    }
+    output {
+        File output_vcf = "~{output_vcf_name}"
+        File output_vcf_index = "~{output_vcf_name}.tbi"
+        File monitoring_log = "monitoring.log"
+    }
+}
+
+task CompareToGroundTruth {
+  input {
+    String left_sample_name
+    String right_sample_name
+    File input_vcf
+    File input_vcf_index
+
+    String input_vcf_name
+
+    File gtr_vcf
+    File gtr_vcf_index
+    File gtr_highconf
+
+    File? interval_list
+    File runs_file
+    File ref_fasta
+    File ref_dict
+    File ref_fasta_sdf
+    String flow_order
+    Array[File] annotation_intervals
+    Int disk_size
+    String docker = "gcr.io/terra-project-249020/jukebox_vc:test_jc_optimize_3d7509"
+  }
+
+  String used_flow_order = (if flow_order=="" then "TACG" else flow_order)
+
+  command <<<
+    set -e
+
+    source ~/.bashrc
+    conda activate genomics.py3
+
+    python -m tarfile -e ~{ref_fasta_sdf} ~{ref_fasta}.sdf
+
+    run_comparison_pipeline.py \
```

Comment:
Perhaps even specific tasks for calculating metrics or performing comparisons could work through a more generic interface. Maybe a catch-all argument for specific arguments like the flow order? Again, OK for future work.